tests/operators/models/test_inversion_recovery.py

```diff
@@ -3,6 +3,7 @@
 import pytest
 import torch
 from mrpro.operators.models import InversionRecovery
+from tests.helper import autodiff_test
```

Comment:
we could consider importing all these in `tests/__init__.py`
Such that we can just do from tests import ..., similar as we do with the randomgenerator. 
I think we realistically only ever import from 'helper' of tests anyways...